sdks/python/apache_beam/yaml/main.py

```diff
@@ -43,8 +44,8 @@ def _pipeline_spec_from_args(known_args):
     raise ValueError(
         "Exactly one of pipeline_spec or pipeline_spec_file must be set.")
   elif known_args.pipeline_spec_file:
-    with open(known_args.pipeline_spec_file) as fin:
```

Comment:
I think we still want to use the with syntax to make sure it gets closed properly.

Reply:
Fixed.